### PR TITLE
cnFilter - Return false if uniqueid

### PR DESCRIPTION
**Pull request by jocteau**
This could cause side effect in the future, if one day we decide to filter some objects with the key `__uniqueid` but is needed for: https://github.com/citizennet/cn-tags-input/pull/22

### CODE REVIEW

File: src/cn-filters.js [hunk 1]
```diff
@@ -27,6 +27,9 @@
           // No substring matching against `undefined`
           return false;
         }
+        if (actual.toString().startsWith('__uniqueid')) {
```
Comment (jocteau) [inline src/cn-filters.js:30]: I'm not sure the `toString` is mandatory though.

**Review by matt-morales — APPROVED** [review]
👍

**Review by davezuch — APPROVED** [review]
I don't really know/remember what this is for, but if it works for you, then I'm good with it.

As an aside, I think it might be worth combining all these AngularJS libraries (`cn-forms`, `cn-flex-form`, `cn-batch-forms`, `angular-schema-form`, `cn-ui`, `cn-tags-input`, `cn-util`, `cn-datetimepicker`, `ui-kit`) into one repo. It doesn't seem helpful at all to have them spread out across so many repos, it just makes more work when you have to make updates to more than one of them. 

I would say bring them all into `wildcat`, but `quicksight` uses them too.

**Comment by jocteau** [comment]
> I don't really know/remember what this is for, but if it works for you, then I'm good with it.
> 
> As an aside, I think it might be worth combining all these AngularJS libraries (`cn-forms`, `cn-flex-form`, `cn-batch-forms`, `angular-schema-form`, `cn-ui`, `cn-tags-input`, `cn-util`, `cn-datetimepicker`, `ui-kit`) into one repo. It doesn't seem helpful at all to have them spread out across so many repos, it just makes more work when you have to make updates to more than one of them.
> 
> I would say bring them all into `wildcat`, but `quicksight` uses them too.

I totally agree. Maybe we should move all these repositories inside a `cn-frontend-utilities` package. That would make everything easier for sure

**Pull request merged**